src/discord/discord_state.rs

```diff
@@ -117,18 +112,6 @@ pub trait DiscordOperations {
     ) -> Result<PooledConnection<'a, DieselConnectionManager>, RunError<ConnectionError>>;
 
     async fn get_player_pfp(&self, p: &Player) -> Result<Option<String>, NMGLeagueBotError>;
-
-    async fn get_guild_scheduled_events(
```

Comment:
this stuff was in the main "discord state" object, which is specific to the main discord. they got pulled out to the helper bot object instead.